src/mainwin.cc

```diff
@@ -2757,8 +2757,13 @@ void MainWindow::hideEditor()
 {
 	if (viewActionHideEditor->isChecked()) {
 		editorDock->close();
+		QAction *exportSTL = editortoolbar->actions().at(12);
```

Comment:
I think that should be needed. The action is globally available in the MainWindow class via `this->fileActionExportSTL` (see the connect call using that).

Reply:
Noted :)